extend/building-components.md

```diff
@@ -67,7 +67,7 @@ Components must be registered by overriding the `registerComponents` method with
 public function registerComponents()
 {
     return [
-        Igniter\HelloWorld\Components\Block::class => [
+        'Igniter\HelloWorld\Components\Block' => [
```

Comment:
Issue here might be the missing leading slash and not the quotes? Try `\Igniter\HelloWorld\Components\Block::class`

Reply:
Update the PR if it works